tests/unit/utils/resource-helper-test.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember';
+import sinonTest from 'ember-sinon-qunit/test-support/test';
+import { module } from 'qunit';
+
+import loadAssets from 'ember-cli-bundle-loader/utils/load-assets';
+import ResourceHelper from 'ember-cli-bundle-loader/utils/resource-helper';
+
+
+module('Unit | Utility | resource loader');
+
+sinonTest('loadJavascript adds the appropriate script tag to the DOM', function (assert) {
```

Comment:
These tests don't rely on sinon, making them regular test is a good indication that they don't mock things and they actually modify the DOM (maybe the should even be integration tests instead).